cmake/alpakaCommon.cmake

```diff
@@ -548,19 +548,18 @@ if(alpaka_ACC_SYCL_ENABLE)
 
         # Enable device-side printing to stdout
         cmake_dependent_option(alpaka_SYCL_ENABLE_IOSTREAM "Enable device-side printing to stdout" OFF "alpaka_ACC_SYCL_ENABLE" OFF)
-        if(BUILD_TESTING)
-            set(alpaka_SYCL_ENABLE_IOSTREAM ON CACHE BOOL "Enable device-side printing to stdout" FORCE)
-        endif()
 
         alpaka_set_compiler_options(HOST_DEVICE target alpaka "-fsycl")
         target_link_options(alpaka INTERFACE "-fsycl")
         alpaka_set_compiler_options(HOST_DEVICE target alpaka "-sycl-std=2020")
+        alpaka_set_compiler_options(HOST_DEVICE target alpaka "-Wno-unused-lambda-capture")
+        alpaka_set_compiler_options(HOST_DEVICE target alpaka "-Wno-unused-parameter")
```

Comment:
Not a fan of disabling compiler warnings globally. if they just occur inside oneAPI headers, can we guard the headers in the C++ code instead?

Reply:
Since they are inside an `if(alpaka_ACC_SYCL_ENABLE)`, aren't they already for oneAPI only? Maybe I'm missing something here

Reply:
Sure, sorry for the confusion. I meant that I would like to have the warnings enabled for alpaka. And if these warnings are triggered in the alpaka code, we should fix them there. If the warnings are triggered inside the headers of oneAPI, we can add the corresponding `#pragma`s before and after the inclusion of the offending oneAPI header.

Reply:
ok, they are triggered by the way SYCL kernels are implemented now to take into account the warp size. I'll try to move them in the alpaka code or see if I can change something in the SYCL kernels.

Reply:
I see. Can we suppress the warnings in the alpaka TaskKernel implementation then? And if not, I guess you can leave the changeset as is. Maybe add a comment that those warnings come from the SYCL kernel implementation.